.github/workflows/test.yml

```
@@ -93,6 +93,19 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
+      # This is a temporary solution.
+      # Once ubuntu-latest includes the latest libxml2 version, this step can be removed.
+      - name: Install homebrew
+        if: ${{ matrix.os == 'ubuntu-latest' && matrix.php == '8.4' }}
+        run: |
+          export HOMEBREW_NO_ANALYTICS=1
+          echo '/home/linuxbrew/.linuxbrew/bin' >> $GITHUB_PATH
+          /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

@jrfnl I'm not sure how to fix the lint error.  Should I use double-quote in this line
```suggestion
        run: |
          export HOMEBREW_NO_ANALYTICS=1
          echo "/home/linuxbrew/.linuxbrew/bin" >> $GITHUB_PATH
          /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

---

`$GITHUB_PATH` needs to be double quoted.